packages/@lwc/ssr-compiler/src/__tests__/utils/expected-failures.ts

```diff
@@ -26,6 +26,9 @@ export const expectedFailures = new Set([
     'scoped-slots/mixed-with-light-dom-slots-inside/index.js',
     'scoped-slots/mixed-with-light-dom-slots-outside/index.js',
     'slot-forwarding/scoped-slots/index.js',
+    'slot-forwarding/slots/mixed/index.js',
+    'slot-forwarding/slots/dangling/index.js',
+    'slot-forwarding/slots/light/index.js',
     'slot-not-at-top-level/advanced/lwcIf/light/index.js',
```

Comment:
There's still an issue with the comments not appearing correctly with light DOM slots.

These tests are still failling to render the HTML bookends in the same way as the runtime.

Reply:
There's also another issue where the `slot` attribute is preserved in some cases, that's not happening in the SSR environment.

These can be observed in:

- `slot-forwarding/slots/mixed/index.js`
- `slot-forwarding/slots/dangling/index.js`

Reply:
I created #5000 to track the HTML bookend issue